test/_e2e/cmd_build_test.go

```diff
@@ -24,3 +31,20 @@ func Build(t *testing.T, knFunc *TestShellCmdRunner, project *FunctionTestProjec
 	project.IsBuilt = true
 
 }
+
+// TestBuild_S2I runs `func build` using the S2I builder.
+func TestBuild_S2I(t *testing.T) {
+	var (
+		root        = "testdata/e2e/testbuild"
+		bin, prefix = bin()
+		cleanup     = Within(t, root) // TODO: replace with Fromtemp?
+		cwd, _      = os.Getwd()
+	)
+	defer cleanup()
+
+	run(t, bin, prefix, "create", "-v", "--language=node", cwd)
```

Comment:
Did you test with typescript? I don't know if s2i can seamlessly do that or not, but it sure would be nice

Reply:
I have not run a typescript test yet, but I am hopeful as well.